### ISSUE TITLE
make stem more flexible

### ISSUE THREAD

**Issue by kepper** [music-encoding/music-encoding#1159]
right now, the class system around the `<stem>` element is very strictly tied to the `MEI.mensural` module. That is fine, as long as `<stem>` is only available in that module. However, it seems foreseeable that other repertoires may want to use that element as well, in which case the classes should be cleaned up, and spread across different modules. This is not an immediate issue, just a reminder to do that maintenance work at some point.

**Comment by ahankinson** [comment]
I'm not sure about this.

`<stem>` was added specifically to [meet mensural-specific needs](https://github.com/music-encoding/music-encoding/commit/9268bfea844fd9258b0b5c620af42d0f68526f17), and we have so far resisted breaking apart `<note>` into their constituent components (stems, notehead, flags). 

So I think `<stem>` should stay in Mensural until such time we decide that notes can be broken apart, which is a much larger discussion.

**Comment by kepper** [comment]
I agree that this will require thorough consideration, and it's not something that we'll want to do in a rush. I'll kick it down the road to _someday_…

**Comment by rettinghaus** [comment]
I'm all in for this, especially looking through the Verovio glasses, where we generate stem elements from CMN stem attributes and have to deal with stem elements for mensural, and the whole thing is quite a mess right now …

**Comment by rettinghaus** [comment]
But I'm with you, we don't have to rush things here.
